newhelm/runners/simple_test_runner.py

```diff
@@ -59,11 +59,13 @@ def run_prompt_response_test(
     for key, annotator in keyed_annotators:
         annotations: List[Annotation] = []
         for interactions_for_item in item_interactions:
-            annotations.append(
-                Annotation.from_instance(
-                    annotator.annotate_test_item(interactions_for_item.interactions)
+            try:
+                annotation = annotator.annotate_test_item(
+                    interactions_for_item.interactions
                 )
-            )
+            except Exception as e:
+                raise Exception(f"Exception while handling: {interactions_for_item}")
```

Comment:
Would adding `from` add any additional info from the raise?

```
raise Exception(f"Exception while handling: {interactions_for_item}") from e
```

Reply:
The best I can tell, using `from` replaces

```
During handling of the above exception, another exception occurred:
```

with

```
The above exception was the direct cause of the following exception:
```

Which is slightly clearer. Added.